openreview/tools.py

```diff
@@ -1718,7 +1718,7 @@ def get_conflicts(author_profiles, user_profile, policy='default', n_years=5):
     conflicts.update(author_domains.intersection(user_info['domains']))
     conflicts.update(author_relations.intersection(user_info['emails']))
     conflicts.update(author_emails.intersection(user_info['relations']))
-    conflicts.update(author_emails.intersection(user_info['emails']))
+    # conflicts.update(author_emails.intersection(user_info['emails']))
```

Comment:
I believe we need to keep doing this for profiles that contain enterprise email addresses like gmail.com, outlook.com where the domains are ignored.

Reply:
Okay, do we need to calculate all of them, if we have already found a conflict?